setup.py

```diff
@@ -60,8 +60,9 @@ def get_extras_require():
         ],
         'example': [
             'chainer', 'keras', 'catboost', 'lightgbm', 'scikit-learn',
-            'tensorflow', 'mxnet', 'xgboost', 'torch', 'torchvision',
-            'dask-ml', 'dask[dataframe]'
+            'mxnet', 'xgboost', 'torch', 'torchvision', 'dask-ml', 'dask[dataframe]',
+            # TODO(Yanase): Update examples to support TensorFlow 2.0.
```

Comment:
How about including the issue link (i.e., `https://github.com/pfnet/optuna/issues/565` ) in this comment?

Reply:
Thanks!
I fixed it: https://github.com/pfnet/optuna/pull/564/commits/067290437303bd6951c911e9cf7a51bf37e4fb30.